### PR TITLE
Added Tender app

### LINKED ISSUE #111: Create app for Tenders
Create an app for tenders
It should display a table-list view of linked pdfs at `/tenders`.

### LINKED ISSUE #115: Fix Page Title for People and Announcement Apps

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #111 
Fixes #115
Deployed : http://13.126.164.61:8000/
Username : test
Password : test

### CODE REVIEW

File: wsgi/odyssy/people/templates/people.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {% extends 'basic/base.html' %}
+{% block title %}
+    IIITV | People
```
Comment (Monal5031) [inline wsgi/odyssy/people/templates/people.html]: It should be `{{ status }}`

File: wsgi/odyssy/people/templates/person.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {% extends 'basic/base.html' %}
+{% block title %}
+    IIITV | {{ status }}
```
Comment (Monal5031) [inline wsgi/odyssy/people/templates/person.html]: `{{ status }}` --> `{{ person.name }}`

File: wsgi/odyssy/tender/models.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        verbose_name = 'Tender'
+        verbose_name_plural = 'Tenders'
+
+    start_date = models.DateTimeField(default=timezone.now)
+    end_date = models.DateTimeField(default=timezone.now)
+    title = models.CharField(max_length=200)
+    slug = models.SlugField(max_length=100, blank=True, null=True)
+    tender = models.FileField(upload_to='tender/')
+    tags = TaggableManager(blank=True)
+
```
Comment (Monal5031) [inline wsgi/odyssy/tender/models.py]: add a `__str__()` to show some name to the objects in admin menu.

File: wsgi/odyssy/basic/templates/basic/base.html [hunk 4]
```diff
@@ -152,6 +152,7 @@ <h5 class="primary-color">Vadodara</h5>
 
                             <li><a href="/careers/careers.html">Career</a></li>
                             <li><a href="/gallery/">Photo Gallery</a></li>
+                            <li><a href="/tender/">Tender</a></li>
```
Comment (mohitkyadav) [inline wsgi/odyssy/basic/templates/basic/base.html]: Shouldn't it be "Tenders"?

**Comment by aashutoshrathi** [comment]
@Monal5031 Pull the changes to AWS.

**Comment by Monal5031** [comment]
@aashutoshrathi Done, please check it once.

**Comment by Monal5031** [comment]
@yashLadha @Prakash2403 @mohitkyadav @geekyJock8 Please review.

**Pull request merged**